starter-code/iteration3/src/app/components/champions/champions.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="container-champions">
+
+  <div *ngFor="let champion of champions">
+    <h1>{{champion.year}}</h1>
+    <img src="{{champion.img}}" alt="" class="logo">
+    <h3>Best Players</h3>
+    <div *ngFor="let player of champion.bestPlayers">
```

Comment:
You would normally add the ngFor on the li element or else you will be creating multiple unordered lists